### PR TITLE
Update ansible submodule to include the calico manifest changes

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
To include submodule updates for calico manifest https://github.com/ppc64le-cloud/k8s-ansible/pull/38 and https://github.com/ppc64le-cloud/k8s-ansible/pull/39

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -24,6 +24,8 @@ require (
 	github.com/kballard/go-shellquote v0.0.0-20180428030007-95032a82bc51 // indirect
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
+	github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749 // indirect
```
Comment (mkumatag) [inline go.mod:27]: but I see some changes here in the go.mod files as well?

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (Rajalakshmi-Girish) [reply go.mod:27]: While I ran `UPDATE_SUBMODULE=true ./hack/build.sh`, it threw:
```
+ go generate ./data
assets_generate.go:11:2: no required module provides package github.com/shurcooL/vfsgen; to add it:
        go get github.com/shurcooL/vfsgen
data/gen.go:3: running "go": exit status 1
```
then doing a `go get github.com/shurcooL/vfsgen` made these changes to go.mod and go.sum

Comment (mkumatag) [reply go.mod:27]: I think we should write some CI jobs for this repo for building, verifying etc..

**Review by mkumatag — APPROVED** [review]

**Pull request merged**